requirements.txt

```diff
@@ -436,24 +350,16 @@ idna==3.10 \
 importlib-metadata==4.8.2 \
     --hash=sha256:53ccfd5c134223e497627b9815d5030edf77d2ed573922f7a0b8f8bb81a1c100 \
     --hash=sha256:75bdec14c397f528724c1bfd9709d660b33a4d2e77387a3358f20b848bb5e5fb
-    # via
-    #   keyring
-    #   markdown
+    # via markdown
 inflection==0.5.1 \
     --hash=sha256:1a29730d366e996aaacffb2f1f1cb9593dc38e2ddd30c91250c6dde09ea9b417 \
     --hash=sha256:f38b2b640938a4f35ade69ac3d053042959b62a0f1076a5bbaa1b9526605a8a2
     # via
     #   -r requirements.in
     #   drf-spectacular
-jeepney==0.7.1 \
-    --hash=sha256:1b5a0ea5c0e7b166b2f5895b91a08c14de8915afda4407fb5022a195224958ac \
-    --hash=sha256:fa9e232dfa0c498bd0b8a3a73b8d8a31978304dcef0515adc859d4e096f96f4f
-    # via
-    #   keyring
-    #   secretstorage
-jira==3.1.1 \
-    --hash=sha256:200c4d19f8be5ae39da3578597b78e21f3db69520c354be782bee9bd8bf21d09 \
-    --hash=sha256:e2fde55d04a421c590cb197cf6b2d01176028004387d3e4cedff653dba408238
+jira==3.8.0 \
```

Comment:
Seems that they are having problems with the release process for a couple of months now https://github.com/pycontribs/jira/issues/1922 , the latest version is `v3.9.4` but is not published to PyPi https://github.com/pycontribs/jira/releases/tag/3.9.4

Not sure if this is something we need to consider

Reply:
Could be worth upgrading if it gets published soon, although looking at the changelog I don't see any major fixes that would make that a priority. The main reason I upgraded was to be able to use the method to delete remote links.